platform/broadcom/sai.mk

```diff
@@ -12,7 +12,7 @@ $(eval $(call add_derived_package,$(BRCM_XGS_SAI),$(BRCM_XGS_SAI_DEV)))
 $(BRCM_XGS_SAI_DEV)_URL = "$(LIBSAIBCM_XGS_URL_PREFIX)/$(BRCM_XGS_SAI_DEV)"
 
 # SAI module for DNX Asic family
-BRCM_DNX_SAI = libsaibcm_dnx_$(LIBSAIBCM_DNX_VERSION)_amd64.deb
+BRCM_DNX_SAI = libsaibcm_$(LIBSAIBCM_DNX_VERSION)_amd64.deb
```

Comment:
the name changed from `libsaibcm_dnx` to `libsaibcm` ?

Reply:
hi @arlakshm yes, this name is changed to match our azure storage name, for SAI 10.x uploaded to azure storage, the DNX sai names changed to this. it is still under ./dnx directory to differentiate with XGS